File: core/src/main/java/com/exozet/android/core/interfaces/DispatchTouchEvent.java
package com.exozet.android.core.interfaces;

import android.view.MotionEvent;

import androidx.annotation.NonNull;

/**
 * Created by Jan Rabe on 28/09/15.
 */
public interface DispatchTouchEvent {

    boolean dispatchTouchEvent(@NonNull final MotionEvent event);
}

File: core/src/main/java/com/exozet/android/core/interfaces/DispatchTouchEvent.kt
package com.exozet.android.core.interfaces

import android.view.MotionEvent

/**
 * Created by Jan Rabe on 28/09/15.
 */
@Deprecated("use DispatchTouchEventHandler", ReplaceWith("DispatchTouchEventHandler"))
interface DispatchTouchEvent {

    fun dispatchTouchEvent(event: MotionEvent): Boolean
}

